lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java

```diff
@@ -56,8 +60,35 @@ public class RemoteCentralRepositorySearcher {
 	public static boolean disableCentralSearch = Boolean
 			.parseBoolean(System.getProperty(RemoteCentralRepositorySearcher.class.getName() + ".disableCentralSearch"));
 
+	public static int cacheExpirationTimeoutMinutes = Integer
+			.parseInt(System.getProperty(RemoteCentralRepositorySearcher.class.getName() + ".cacheExpirationTimeoutMinutes", "-1"), 10);
+
 	private OkHttpClient client = new OkHttpClient();
 
+	private static class CacheManager<K,V> {
+		private static long DEFAULT_EXPIRATION_TIMEOUT = 30; 
+
+		private Cache <K, V> cache;
+		private long cacheExpirationTimeout = DEFAULT_EXPIRATION_TIMEOUT;
+		
+		public CacheManager(long expirationTimeoutMinutes) {
```

Comment:
It is exposed as environment variable: https://github.com/eclipse/lemminx-maven/blob/71b92d76055630678336989a05db71550e39ebc1/lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java#L63-L64 at the moment.

Reply:
IMHO environement variable is little boring for the user, having an UI settings should be better.

Reply:
User shouldn't have to deal with such implementation details. A system property is already good enough. Maybe later it can be exposed differently, but we're still far from needing it.